### ISSUE TITLE
akka-management and CoordinatedShutdown

### ISSUE THREAD

**Issue by sebarys** [akka/akka-management#291]
Hi,

I have a question: do I need to manually stop akka-management in my custom Task when using akka coordinated shutdown extension ( https://doc.akka.io/docs/akka/2.5.6/scala/actors.html#coordinated-shutdown )?

Additionally I found in akka-management documentation sentence: "In a dynamic environment you might stop instances of Akka Management, for example if you don’t want to free up resources taken by the HTTP server serving the Management routes." ( https://developer.lightbend.com/docs/akka-management/current/akka-management.html#stopping-akka-management )
Should not it be rather "if you want to free up" instead of "if you don’t want to free up" ?

Thanks,
sebarys

**Comment by chbatey** [comment]
Yep new nodes will need the http endpoints for bootstrap so we can fix the docs under this issue

**Comment by glammers1** [comment]
btw, the negation was fixed here #334 https://github.com/akka/akka-management/pull/334/files#diff-4fec015fc27c67d13e5612cfbc1dcfd5R125
